docs/startup/preview/index.md

```diff
@@ -0,0 +1,530 @@
+
+Alternatively, you also can use the `MessageStream#mergeAll` static method to merge MessageStreams without operating on an initial stream.
+
+### SendTo
+Sends all messages from this MessageStream to the provided OutputStream. You can specify the key and the value to be used for the outgoing message.
+
+{% highlight java %}
+// Output a new message with userId as the key and region as the value to the “user-region” stream.
+MessageStream<PageView> pageViews = ...
+OutputStream<String, String, PageView> userRegions
+                           = graph.getOutputStream(“user-region”,
+                                                   pageView -> pageView.getUserId(),
+                                                   pageView -> pageView.getRegion())
+pageView.sendTo(userRegions);
+{% endhighlight %}
+
+
+### Sink
+Allows sending messages from this MessageStream to an output system using the provided [SinkFunction](/learn/documentation/{{site.version}}/api/javadocs/org/apache/samza/operators/functions/SinkFunction.html).
+
+This offers more control than sendTo since the SinkFunction has access to the `MessageCollector` and the `TaskCoordinator`. For instance, you can choose to manually commit offsets, or shut-down the job using the TaskCoordinator APIs. This operator can also be used to send messages to non-Samza systems (e.g. remote databases, REST services, etc.)
+
+{% highlight java %}
+// Repartition pageView by userId.
+MessageStream<PageView> pageViews = ...
+pageViews.sink( (msg, collector, coordinator) -> {
+// Construct a new outgoing message, and send it to a kafka topic named TransformedPageViewEvent.
+ collector.send(new OutgoingMessageEnvelope(new SystemStream(“kafka”,
+                         “TransformedPageViewEvent”), msg));
+} )
+{% endhighlight %}
+
+### Join
+
+The Join operator joins messages from two MessageStreams using the provided pairwise [JoinFunction](/learn/documentation/{{site.version}}/api/javadocs/org/apache/samza/operators/functions/JoinFunction.html). Messages are joined when the keys extracted from messages from the first stream match keys extracted from messages in the second stream. Messages in each stream are retained for the provided ttl duration and join results are emitted as matches are found.
+
+{% highlight java %}
+// Joins a stream of OrderRecord with a stream of ShipmentRecord by orderId with a TTL of 20 minutes.
+// Results are produced to a new stream of FulfilledOrderRecord.
+MessageStream<OrderRecord> orders = …
+MessageStream<ShipmentRecord> shipments = …
+
+MessageStream<FulfilledOrderRecord> shippedOrders = orders.join(shipments, new OrderShipmentJoiner(), Duration.ofMinutes(20) )
+
+// Constructs a new FulfilledOrderRecord by extracting the order timestamp from the OrderRecord and the shipment timestamp from the ShipmentRecord.
+ class OrderShipmentJoiner implements JoinFunction<String, OrderRecord, ShipmentRecord, FulFilledOrderRecord> {
```

Comment:
I'm not sure I understand. I don't see a spelling error. What is this comment suggesting?